params/config.go

```diff
@@ -313,6 +313,8 @@ type NodeConfig struct {
 	// NetworkID sets network to use for selecting peers to connect to
 	NetworkID uint64 `json:"NetworkId" validate:"required"`
 
+	RootDataDir string `json:"-"`
+
```

Comment:
@richard-ramos I know this is a very old PR, but do you remember why you added `RootDataDir` with a `json:"-"` tag? Was it a security reason or something else?

Reply:
Sadly, the reasons for this field not being exported when marshalled were lost in the mists of time

Reply:
<img src="https://media.tenor.com/DnpJz0JZo9MAAAAC/gandalf.gif" />